tests/Unit/Infrastructure/EventHandling/DomainEventHandler/BucketLoggingHandlerTest.php

```diff
@@ -86,4 +121,12 @@ public function getBuckets(): array {
 		$campaign->addBucket( new Bucket( 'test1', $campaign, true ) );
 		return $campaign->getBuckets();
 	}
+
+	private function createBucketLoggingHandler( EventDispatcher $eventDispatcher, BucketLogger $logger ): BucketLoggingHandler {
```

Comment:
I like that you did not include the consent state in the params - while that adds a line to the test setup, it makes it really explicit what the consent is.